Description: Fixes to Romanian language style
 This patch fixes several issues in the Romanian version of latex2html.
Author: Roland Stigge <stigge@antcom.de>
Bug-Debian: http://bugs.debian.org/119530

--- latex2html-2008-debian1.orig/styles/romanian.perl
+++ latex2html-2008-debian1/styles/romanian.perl
@@ -16,23 +16,23 @@ package main;
 
 if (defined &addto_languages) { &addto_languages('romanian') };
 
-&do_require_extension ('latin2');
+&do_require_extension ('latin10');
 
 sub romanian_titles {
     $toc_title = "Cuprins";
-    $lof_title = "List&auml; de figuri";
-    $lot_title = "List&auml; de tabele";
+    $lof_title = "List&#103; de figuri";
+    $lot_title = "List&#103; de tabele";
     $idx_title = "Glosar";
     $ref_title = "Bibliografie";
     $bib_title = "Bibliografie";
     $abs_title = "Rezumat";
     $app_title = "Anexa";
-    $pre_title = "Prefa&tcedil;&auml;";
+    $pre_title = "Prefa&#21B;&#103;";
 ##    $foot_title = "";
 ##    $thm_title = "";
     $fig_name = "Figura";
     $tab_name = "Tabela";
-    $prf_name = "Demonstra&tcedil;ie";
+    $prf_name = "Demonstra&#21B;ie";
     $page_name = "Pagina";
   #  Sectioning-level titles
     $part_name = "Partea";
@@ -52,7 +52,7 @@ sub romanian_titles {
 ##    $prev_name = "";
 ##    $group_name = "";
   #  mail fields
-    $encl_name = "Anex&auml;";
+    $encl_name = "Anex&#103;";
     $headto_name = "Pentru";
     $cc_name = "Copie";
 
@@ -75,7 +75,7 @@ sub romanian_today {
 &romanian_titles;
 $default_language = 'romanian';
 $TITLES_LANGUAGE = 'romanian';
-$romanian_encoding = 'iso-8859-2';
+$romanian_encoding = 'is-8859-16';
 
 # $Log: romanian.perl,v $
 # Revision 1.1  1998/08/25 01:59:07  RRM

